liquibase-dist/src/main/install4j/liquibase.install4j

```diff
@@ -8,7 +8,8 @@
     <variables>
       <variable name="liquibaseVersion" value="0-SNAPSHOT" />
     </variables>
-    <codeSigning macEnabled="true" macPkcs12File="../../../target/keys/datical_apple.p12" windowsEnabled="true" windowsPkcs12File="../../../target/keys/datical_windows.pfx" macNotarize="true" appleId="daticaldev@datical.com" useAscProvider="true" ascProvider="BQ3PYUB226">
+    <codeSigning macPkcs12File="../../../target/keys/datical_apple.p12" windowsEnabled="true" windowsPkcs12File="../../../target/keys/datical_windows.pfx" macNotarize="true">
```

Comment:
I think we can manually remove those 2 mac properties from here so they don't cause any confusion in the future.

Reply:
changes made : verify again